src/Fields/interpolate.jl

```diff
@@ -68,30 +68,30 @@ end
     x₀ = xnode(1, 1, 1, grid, locs...)
     Δx = xspacings(grid, locs...)
     FT = eltype(grid)
-    return convert(FT, (x - x₀) / Δx) # 0 - based indexing!
+    return convert(FT, (x - x₀) / Δx) + 1 # 0 - based indexing so we add 1
```

Comment:
```suggestion
    return convert(FT, (x - x₀) / Δx) + 1 # 1-based indexing
```

Just a nitpick, but I think it's better to simply state that this is 1-based